### ISSUE TITLE
Won't Install (is it me?)

### ISSUE THREAD

**Issue by GitStevie** [geerlingguy/temperature-monitor#31]
I may be an idiot as I am a Newbie, but I have a catastrophic error and a question.  The error is when I try to run the command "ansible-galaxy install -r requirements.yml" from the playbooks folder I get:  

```
downloading role '---', owned by
Sorry, --- was not found on galaxy.ansible.com.
 downloading role 'git', owned by -%20src%3A%20geerlingguy
Sorry, - src: geerlingguy.git was not found on galaxy.ansible.com.
 downloading role 'mysql', owned by -%20src%3A%20geerlingguy
Sorry, - src: geerlingguy.mysql was not found on galaxy.ansible.com.
 downloading role 'nodejs', owned by -%20src%3A%20geerlingguy
Sorry, - src: geerlingguy.nodejs was not found on galaxy.ansible.com.
 downloading role 'raspberry-pi', owned by -%20src%3A%20geerlingguy
Sorry, - src: geerlingguy.raspberry-pi was not found on galaxy.ansible.com.
```
The question is:  Are Vagrant and virtual-box required?  I am running multiple Raspberrys and don't need to run virtual systems on one box as far as I know.  

I've got my sensors all wired up and running fine - I'd love to get this software figured out!  Thank you.  Steve

**Comment by geerlingguy** [comment]
Check if you have internet access at all from whatever device you're using to run the ansible-galaxy command. Usually those errors would only appear if you have no internet, or if your internet connection is proxied or firewalled in some way.

You might want to ask about these errors in the #ansible channel on Freenode IRC, or in the Ansible Project google group.

**Comment by GitStevie** [comment]
Problem solved:  as geerlingguy mentioned, ansible has to be version 2.0 or higher.  When I installed this it put on V 1.7x.  I updated ansible and brought it to v 2.2.x

Now I'm passed this hurdle and on to the next Newbie showstopper, lol!

Look here for instructions on how to upgrade ansinble on the Pi
[https://arnesonium.com/2016/11/installing-ansible-2-2-0-on-a-raspberry-pi/](url)

Thanks for your help!
